### PR TITLE
Add temperature smoothing to EFAS data conditioning script

### LINKED ISSUE #61: Extend EFAS smoothing script to include temperature
## Reason
As noted in previous efforts with the displacement data, raw temperature data as collected from the Dr. Sinter EFAS machine can be quite noisy. Large spikes in the temperature ramp, as well as smaller spikes in the hold period, can provide erroneous variations when these temperature data are used for simulation inputs.

## Design
Copy the data smoothing and saving procedures implemented for the displacement for use on the temperature data. 

## Impact
Ease of data smoothing for validation studies, where spikes in the data may significantly disrupt simulations without good physical case; extends the existing capability of the script

### PR DESCRIPTION
**Pull request by sapitts**
Also attempts to update the bad lines behavior to skip, from default error, and includes auto-formatting changes from vscode
Closes #61

### CODE REVIEW

File: scripts/drsinter_data_extraction.py [hunk 1]
```diff
@@ -215,18 +253,19 @@ def isNaN(string):
...
 metadata.loc[0] = ['Data Sampling Rate (s)', sampling_rate]
 
-### Get channel names (with NaN for missing column entries)
+# Get channel names (with NaN for missing column entries)
 channel_row = raw_data[0].loc[raw_data[0][dummy_names[0]] == 'Waveform Name']
 
 header_print('DATA UNITS')
 
-### Get units for each channel (TODO: remove weird non-standard character in front of C for temp)
+# Get units for each channel (TODO: remove weird non-standard character in front of C for temp)
```
Comment (cticenhour) [inline scripts/drsinter_data_extraction.py]:
```suggestion
# Get units for each channel
```

I must have neglected to remove this TODO when I added removal of non-ASCII characters. If you agree, we should probably remove part of this comment.

File: scripts/drsinter_data_extraction.py [hunk 2]
```diff
@@ -367,14 +434,15 @@ def isNaN(string):
...
         ax1.relim()
         ax1.autoscale_view()
         for line in ax1.lines:
-            line.set_marker(None)
+            line.set_marker('')
         ax1.get_legend().remove()
         plt.tight_layout(pad=2)
-        plt.savefig(cols[i] + plot_format)
+        plt.savefig(name_base + cols[i] + plot_format)
         print('Exported', cols[i] + plot_format)
```
Comment (cticenhour) [inline scripts/drsinter_data_extraction.py]:
```suggestion
        print('Exported', name_base + cols[i] + plot_format)
```
In order to match line above.

**Review by cticenhour — CHANGES_REQUESTED** [review]
Only a couple comments to resolve. Thanks for improving the script!

File: scripts/drsinter_data_extraction.py (hunk 1 above)
Comment (sapitts) [reply scripts/drsinter_data_extraction.py]: added into second commit

File: scripts/drsinter_data_extraction.py (hunk 2 above)
Comment (sapitts) [reply scripts/drsinter_data_extraction.py]: added into the second commit

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
Thanks again @sapitts!

**Pull request merged**